modules/migration/migration-5.0.0_to_5.1.0/wso2-is-migration-client/src/main/java/org/wso2/carbon/is/migration/MigrationDatabaseCreator.java

```diff
@@ -228,7 +231,7 @@ private void executeSQLScript(String dbscriptName) throws Exception {
             if (sql.length() > 0) {
                 executeSQL(sql.toString());
             }
-        } catch (IOException e) {
+        } catch (Exception e) {
             log.error("Error occurred while executing SQL script for migrating database", e);
             throw new Exception("Error occurred while executing SQL script for migrating database", e);
```

Comment:
removed the log since it should not log and throw at the same time. fixed with https://github.com/wso2/product-is/pull/402